Splunk sid (#23911)
demisto/content: Packs/SplunkPy/ReleaseNotes/3_0_5.md

#### Integrations
##### SplunkPy

> - Added the sid for any **splunk-search** command status for quicker results return.

Reviewer: ```suggestion
- Added the *sid* output to the **splunk-search** command.
```
